platform/mds/mds/src/main/java/org/motechproject/mds/repository/internal/AllEntityDrafts.java

```diff
@@ -68,7 +68,7 @@ public void setProperties(EntityDraft draft, Entity entity, String username) {
 
         draft.setDraftOwnerUsername(username);
         draft.setLastModificationDate(DateUtil.nowUTC());
-
+        draft.setSuperClass(entity.getSuperClass());
```

Comment:
can you change in git history if this line existed in the past ? I'm wondering why it was not here, it always worth to check git history.